src/org/opendatakit/briefcase/ui/automation/AutomationPanel.java

```diff
@@ -2,51 +2,70 @@
 import javax.swing.JPanel;
 import org.bushe.swing.event.annotation.AnnotationProcessor;
 import org.opendatakit.briefcase.automation.AutomationConfiguration;
+import org.opendatakit.briefcase.model.BriefcaseFormDefinition;
+import org.opendatakit.briefcase.model.BriefcasePreferences;
 import org.opendatakit.briefcase.reused.BriefcaseException;
+import org.opendatakit.briefcase.util.FormCache;
 
 public class AutomationPanel {
   public static final String TAB_NAME = "Automation";
 
   private final AutomationPanelForm view;
-  private final AutomationConfiguration configuration;
+  private final FormCache formCache;
+  private final BriefcasePreferences appPreferences;
 
-
-  public AutomationPanel(AutomationPanelForm view, AutomationConfiguration configuration) {
+  public AutomationPanel(AutomationPanelForm view, BriefcasePreferences appPreferences, FormCache formCache) {
     AnnotationProcessor.process(this);
     this.view = view;
-    this.configuration = configuration;
-
-    view.onSelectScriptDir(path -> configuration.setScriptLocation(path));
+    this.formCache = formCache;
+    this.appPreferences = appPreferences;
 
-    view.onGenerate(() -> {
+    view.onGenerate(config -> {
+      System.out.println(config.getScriptLocation());
       if (isWindows())
-        generateScript("automation.bat", "echo %JAVA_HOME%");
+        generateScript("automation.bat",  config);
       else
-        generateScript("automation.sh", "echo $JAVA_HOME");
+        generateScript("automation.sh", config);
     });
   }
 
-  private void generateScript(String scriptName, String commands) {
+  private void generateScript(String scriptName, AutomationConfiguration configuration) {
     Path scriptDirPath = configuration.getScriptLocation().orElseThrow(() -> new BriefcaseException("No script location directory defined"));
-    Path scriptFilePath = Paths.get(scriptDirPath.toString() + scriptName);
+    String storageDir = appPreferences.getBriefcaseDir().orElseThrow(() -> new BriefcaseException("")).getParent().toString();
+    String template = "java -jar briefcase.jar --export --form_id {0} --storage_directory " +
+        storageDir + " --export_directory /tmp --export_filename {1}.csv";
     try {
-      Files.write(scriptFilePath, commands.getBytes());
-    } catch (IOException e) {
-      System.err.println(e.getMessage());
+      PrintWriter writer = new PrintWriter(scriptDirPath.toFile() + File.pathSeparator + scriptName);
+      List<BriefcaseFormDefinition> forms = formCache.getForms();
+      System.out.println(forms.size());
+      System.out.println(storageDir);
+      for (BriefcaseFormDefinition form: forms) {
+        Object[] params = new Object[]{form.getFormId(), form.getFormName()};
+        String command = MessageFormat.format(template, params);
+        writer.println(command);
+        writer.close();
+      }
+      writer.close();
+    } catch (FileNotFoundException e) {
+      e.printStackTrace();
     }
+
```

Comment:
So, I'm missing some things for Mac and Linux hosts:
- Missing the [shebang](https://en.wikipedia.org/wiki/Shebang_(Unix)) in the first line
- Missing POSIX execution permissions (You can set those with `Files.setPosixFilePermissions` and `PosixFilePermissions.fromString()`)

Windows hosts don't need any of that as far as I know